Library/src/Systems/Sequence/SequenceSystem.cpp

```diff
@@ -139,6 +193,17 @@ void SequenceSystem::GetSequence(const String& SequenceKey, SequenceResultCallba
 
 void SequenceSystem::DeleteSequences(const Array<String>& InSequenceKeys, NullResultCallback Callback)
 {
+	for (size_t i = 0; i < InSequenceKeys.Size(); i++)
+	{
+		if (!ValidateKey(InSequenceKeys[i]))
+		{
+			CSP_LOG_FORMAT(csp::systems::LogLevel::Error,
+						   "Cannot delete Sequence. New Key: %s contains invalid characters",
```

Comment:
I don't think the log message should say "New Key:", just "Key:" as it is not a new key.